src/sm64ap.cpp

```diff
@@ -267,7 +271,7 @@ void SM64AP_SetCourseMap(std::map<int,int> map) {
 }
 
 void SM64AP_SetMoveRandoVec(int vec) {
-    for (int i = 0; i < SM64AP_NUM_ABILITIES; i++) {
+    for (int i = 1; i < SM64AP_NUM_ABILITIES; i++) { // Start at 1, DJ bit is unnecessary
```

Comment:
Does this cause issues without this change? If not, keep as is

Reply:
Without these changes, current APWorld will always give Double Jump, as the DJ bit is always set to 0